pages/loginPage.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://splash-ui.vercel.app/CSS/main.css">
+    <link rel="stylesheet" href="/css/loginPage.css">
+    <title>Sneaker Street</title>
+</head>
+<body>
+    <header class="header-main">
+        <nav class="nav nav-dark" style="position: relative;">
+            <a href="/index.html">
+                <h3>SNEAKER STREET</h3>
+            </a>
+            <button class="hamburger">
+                <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
+            </button>
+            <div class="search-container">
+                <input type="text" placeholder="🔍  Search.." name="search">
+            </div>
+            <ul class="nav-list">
+                <li>
+                    <a href="/pages/loginPage.html">
+                        <button class="btn btn-secondary">Login</button>
+                    </a>
+                </li>
+                <li>
+                    <a href="/pages/wishList.html">
+                        <button class="btn btn-icon">
+                            <i class="fa fa-heart fa-lg" aria-hidden="true"></i>
+                        </button>
+                    </a>
+                </li>
+                <li>
+                    <a href="/pages/cartList.html">
+                        <button class="btn btn-icon">
+                            <i class="fa fa-shopping-cart fa-lg" aria-hidden="true"></i>
+                        </button>
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <div class="main-wrapper">
+        <div class="login-wrapper">
+            <div class="login-text">Login</div>
+            <div class="input-area">
+                <form class="form-area">
+                    <div>
+                        <label class="email-label">Email</label>
```

Comment:
Consider using `for` attribute in `label` to link it to its `input` with `id` attribute

Reply:
Got it will make that changes